core-services/prow/02_config/_config.yaml

```diff
@@ -170,6 +170,7 @@ tide:
     - openshift/dedicated-admin-operator
     - openshift/descheduler-operator
     - openshift/elasticsearch-operator
+    - openshift/elasticsearch-proxy
```

Comment:
This only adds merging for release branches -- also add to the other list so you get `master` as well

Reply:
@stevekuznetsov did i update this file correctly?